hyperon_das_atomdb/adapters/hash_table.py

```diff
@@ -619,3 +642,12 @@ def _calculate_composite_type_hash(self, composite_type: list) -> str:
                     _hash_copy
                 )
         return ExpressionHasher.composite_hash(composite_type)
+
+    def _remove_links_not_toplevel(self, matches: list) -> list:
+        matches_copy = matches[:]
+        for match in matches:
+            link_handle = match[0]
+            links = self.db.link.get_arity(len(match[-1]))
```

Comment:
What does this call do? This code seems odd. We should discuss it in our sync.

Reply:
This method returns the links. Because in the RAM only adapter implementation I separated the "tables" by arity (1,2,n). But we can discuss this in sync